modules/supported-log-outputs.adoc

```diff
@@ -17,29 +17,29 @@ Outputs can be any of the following types:
 |Logging versions
 |Supported collector type
```

Comment:
Fluentd should not appear anywhere in the 6.0 docs, but the outputs still need to be documented.